doc/adrs/030_add_a_link_representation.adoc

```diff
@@ -0,0 +1,54 @@
+= ADR-030 - Add a link representation
+
+== Context
+
+We need to have a batter integration with remote systems.
+For that, we want to be able to display links related to the currently selected element.
+We will thus introduce a new representation named link to display those links.
+This representation will be integrated in the right side of the workbench since it is a representation parameterized by the current selection.
+
+== Decision
+
+The link representation will be created by adding support for links in the backend.
+We will not create a new representation from scratch on the backend.
+
+=== Adding link support to forms
+
+We will start by adding support for `Link` to forms.
+Links will be another kind of widget just like checkbox or textfield.
+They will only contain a label and an URL to create the link.
```

Comment:
I'll add something about that in the consequences (the part which leads to the creation of new ADRs) since the addition of tooltips, decorators or things like that would impact all widgets (there's nothing specific to links here).